### ISSUE TITLE
Encryption key with % fails horribly

### ISSUE THREAD

**Issue by tarkatronic** [orcasgit/django-pgcrypto-expressions#1]
This one took me a good while to figure out as I was only seeing this as the relevant piece of my traceback:

```
  File "/Users/jwilhelm/Documents/workspace/django-pgcrypto-expressions/pgcrypto_expressions/fields.py", line 85, in as_sql
    return self.decrypt_sql % sql, params
TypeError: not enough arguments for format string
```

I'm also having difficulty writing a test case specifically to cover this, primarily due to a lack of experience with pytest. However, if I add a `%` to pgcrypto_expressions.test.settings.SECRET_KEY (e.g., `sec%ret`), nearly all of the tests fail.

**Comment by radeksvarz** [comment]
I should have read the issue tracker before spending time debugging :(

The root cause is in the file fields.py - class DecryptedCol.

as_sql tries to include the table name, where also secret string can provide string formatting commands unintentionally:

```
    return self.decrypt_sql % sql, params
```

where in my case: 

```
self.decrypt_sql = '$-41!d%xblej'
```

so indeed this throughs the exception:

```
"pgp_sym_decrypt(%s, '$=m_z9$-41!d%xblej')::text" % "quickstart_mymodel"
```

as it tries to render some hexa number into the string

Carl might want to change how the decrypt_sql is formed in the code.

Currently the workaround is to avoid % in the secret.

**Comment by carljm** [comment]
Thanks for the report! Yeah, writing a good test specifically for this issue was difficult (and I know py.test pretty well!) I ended up writing a fairly complex test and deciding that I didn't really trust it, so I punted and just added the `%` in the default secret key in the tests settings file instead.

Fixed by #3 -- will merge once Travis approves.
